### PR TITLE
meson.build: strip newlines from git output

### PR DESCRIPTION
**Pull request by rchincha**
Fixes issue #4223

Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -308,7 +308,7 @@ if time_epoch == '' and git.found() and run_command('test', '-e', '.git', check:
     # If we're in a git repository, use the creation time of the latest git tag.
     latest_tag = run_command(git, 'describe', '--abbrev=0', '--tags', check: false).stdout().strip()
     if latest_tag != ''
-      time_epoch = run_command(git, 'log', '--no-show-signature', '-1', '--format=%at', latest_tag, check: true).stdout()
+      time_epoch = run_command(git, 'log', '--no-show-signature', '-1', '--format=%at', latest_tag, check: true).stdout().splitlines()[0]
```
Comment (brauner) [inline meson.build:311]: This seems to fail to build on Ubuntu 22.04 with a complaint about `splitlines()` missing.

Comment (rchincha) [reply meson.build]: fixed.

**Comment by rchincha** [comment]
fyi: https://github.com/lxc/lxc/pull/4226 is the same fix to `master` branch

**Comment by brauner** [comment]
Thanks!

**Pull request merged**